pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -865,6 +868,61 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 				if err != nil {
 					return err
 				}
+				if driverName == "pxd" {
+					backupLocation, err := storkops.Instance().GetBackupLocation(backup.Spec.BackupLocation, backup.Namespace)
```

Comment:
Can you make this logic more of a utils call than embedding into applicationbackup and applicationrestore. The check is same is for both only the error reporting is different.
Please make ita utils.API call to avoid duplicate code.